usagov/vote-gov-drupal
# Vote-2495: Add/Update Cypress Tests

## Jira ticket

[Vote-2472](https://cm-jira.usa.gov/browse/VOTE-2472)
[Vote-2695](https://cm-jira.usa.gov/browse/VOTE-2495)

## Description

Adding test to verify the NVRF download on state page 
Also updating the link validation tests to run off the sitemap and not the `.json` file to capture any new pages that are added

## Deployment and testing

### Post-deploy steps

run `lando retune`

### QA/Testing instructions

1. cd into `testing`
2. run `npm run cy:open`
3. select the `nvrf-download` test and verify that test is running 
4. select both `external` and `internal` links test and verify that test is running (you dont need to let it run all the way through unless you want too)

## Checklist for the Developer




## Checklist for the Peer Reviewers
